CHANGES/6205.bugfix

```diff
@@ -0,0 +1,2 @@
+Declared the minimum required version of ``setuptools`` in the :pep:`517`
```

Comment:
I just realized that this is marked as a bugfix. It's not a runtime bug. So I'd probably use `misc` in the file name.

Reply:
changed to misc